src/main.c

```diff
@@ -2437,6 +2437,11 @@ void handle_movement(double dt) {
                 dy = 8;
             }
         }
+        if (glfwGetKey(g->window, CRAFT_KEY_HIGH_FLY)) {
+            if (g->flying){
+                vy = 35;
```

Comment:
A small comment saying what this is actually changing may be nice for clarity, as "vy" is a little vague on what the variable actually is.

Reply:
This variable is pre declared in the code. This is not something which I have added, I have simply used it to implement the functionality.